frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable/ResponsesTable.tsx

```diff
@@ -7,42 +8,89 @@ import {
   useResizeColumns,
   useTable,
 } from 'react-table'
-import { Flex, Table, Tbody, Td, Th, Thead, Tr } from '@chakra-ui/react'
+import {
+  Flex,
+  Icon,
+  Table,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from '@chakra-ui/react'
 
-import { FormResponseMode, SubmissionMetadata } from '~shared/types'
+import {
+  FormResponseMode,
+  SubmissionMetadata,
+  WorkflowStatus,
+} from '~shared/types'
 import { centsToDollars } from '~shared/utils/payments'
 
+import Badge from '~components/Badge'
+
 import { useAdminForm } from '~features/admin-form/common/queries'
+import {
+  MRF_CURRENT_STEP_LABEL,
+  MRF_FIRST_STEP_TIMESTAMP_LABEL,
+  MRF_STATUS_LABEL,
+} from '~features/admin-form/responses/constants'
 
+import { getCurrentStepString } from '../../../../common/utils/mrfSubmissionView'
 import { useUnlockedResponses } from '../UnlockedResponsesProvider'
 
 import { getNetAmount } from './utils'
 
 type ResponseColumnData = SubmissionMetadata
 
-const RESPONSE_TABLE_COLUMNS: Column<ResponseColumnData>[] = [
+const PendingBadge = () => (
+  <Badge
+    width="fit-content"
+    display="flex"
+    textColor="warning.700"
+    backgroundColor="warning.100"
+  >
+    <Icon as={BiSolidHourglass} mr="0.25rem" fontSize="1rem" />
+    <Text textStyle="caption-1">Pending</Text>
+  </Badge>
+)
+
+const CompletedBadge = () => (
+  <Badge
+    width="fit-content"
+    display="flex"
+    textColor="success.700"
+    backgroundColor="success.100"
+  >
+    <Icon as={BiCheckDouble} mr="0.25rem" fontSize="1rem" />
+    <Text textStyle="caption-1">Completed</Text>
+  </Badge>
+)
+
+const NON_MRF_RESPONSE_TABLE_COLUMNS: Column<ResponseColumnData>[] = [
```

Comment:
Would prefer to use "base" rather than "non-MRF". As non-MRF is not an exception. i.e., Payment is special, MRF is special, but non-MRF is not special.

```suggestion
const BASE_RESPONSE_TABLE_COLUMNS: Column<ResponseColumnData>[] = [
```

Reply:
done